dashboard/config/levels/custom/music/musiclab_intro_freeplay.level

```diff
@@ -0,0 +1,35 @@
+<Music>
+  <config><![CDATA[{
+  "published": true,
+  "game_id": 70,
+  "created_at": "2024-01-09T03:44:34.000Z",
+  "level_num": "custom",
+  "user_id": 574,
+  "properties": {
+    "encrypted": "false",
+    "hide_share_and_remix": "false",
+    "background": "music-black",
+    "level_data": {
+      "library": "will",
+      "startSources": {
+        "blocks": {
+          "languageVersion": 0,
+          "blocks": [
+            {
+              "type": "when_run_simple2",
```

Comment:
`deletable: false / movable: false` here too for the when_run block (and all others)?

Reply:
Done.